src/admin/components/forms/field-types/RichText/elements/link/index.scss

```diff
@@ -18,6 +18,7 @@
     }
 
     .popup__scroll {
+      white-space: pre;
```

Comment:
@jmikrut I had to add this rule and I did it here, but it might be something we want to do for all popups. I wasn't sure. The white-space rule is normally `no-wrap` but that was removing whitespace between translated text and the link in the popup.